usl_models/tests/flood_ml/mock_dataset.py

```diff
@@ -77,3 +85,47 @@ def generator():
     if batch_size:
         dataset = dataset.batch(batch_size)
     return dataset
+
+
+def mock_prediction_dataset(
+    params: model_params.FloodModelParams,
+    height: int = 100,
+    width: int = 100,
+    batch_count: int = 1,
+    batch_size: int = 1,
+    n: int = 0,
```

Comment:
Why is the default zero? The docstring says this is optional, so should we default to None instead?

Reply:
Done.